setup.py

```diff
@@ -20,48 +20,51 @@
         "https://github.com/DLR-RM/stable-baselines3.git@master"
     )
 
+# pinned to 0.21 until https://github.com/DLR-RM/stable-baselines3/pull/780 goes
+# upstream.
+GYM_VERSION_SPECIFIER = "==0.21.0"
+
+# Note: the versions of the test and doc requirements should be tightly pinned to known
+#   working versions to make our CI/CD pipeline as stable as possible.
 TESTS_REQUIRE = (
     [
-        "seals",
-        "black[jupyter]",
-        "coverage",
-        "codecov",
-        "codespell",
-        "darglint",
-        "filelock",
-        # TODO(adam): remove pin once flake8-isort fixed:
-        #  https://github.com/gforcada/flake8-isort/issues/115
+        "seals==0.1.2",
```

Comment:
I think seals shouldn't make breaking changes (and if it does we know who to complain to...)

```suggestion
        "seals~=0.1.2",
```

Reply:
I would rather propose releasing seals `1.0.0` then. Having a `0.x.x` release signals, that the API might change at any time.
See the [semver spec](https://semver.org/#spec-item-4) for details.